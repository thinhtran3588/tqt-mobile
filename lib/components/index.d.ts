export * from './text/text.component';
export * from './view/view.component';
export * from './loading/loading.component';
export * from './loading-screen/loading-screen.component';
