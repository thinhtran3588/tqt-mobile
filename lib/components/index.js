Object.defineProperty(exports,"__esModule",{value:true});var _text=require("./text/text.component");Object.keys(_text).forEach(function(key){if(key==="default"||key==="__esModule")return;Object.defineProperty(exports,key,{enumerable:true,get:function get(){return _text[key];}});});var _rootLayout=require("./root-layout/root-layout.component");Object.keys(_rootLayout).forEach(function(key){if(key==="default"||key==="__esModule")return;Object.defineProperty(exports,key,{enumerable:true,get:function get(){return _rootLayout[key];}});});var _loading=require("./loading/loading.component");Object.keys(_loading).forEach(function(key){if(key==="default"||key==="__esModule")return;Object.defineProperty(exports,key,{enumerable:true,get:function get(){return _loading[key];}});});var _loadingScreen=require("./loading-screen/loading-screen.component");Object.keys(_loadingScreen).forEach(function(key){if(key==="default"||key==="__esModule")return;Object.defineProperty(exports,key,{enumerable:true,get:function get(){return _loadingScreen[key];}});});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9jb21wb25lbnRzL2luZGV4LnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJ5REFBQSxpT0FDQSxpUUFDQSxnUEFDQSIsInNvdXJjZXNDb250ZW50IjpbImV4cG9ydCAqIGZyb20gJy4vdGV4dC90ZXh0LmNvbXBvbmVudCc7XG5leHBvcnQgKiBmcm9tICcuL3Jvb3QtbGF5b3V0L3Jvb3QtbGF5b3V0LmNvbXBvbmVudCc7XG5leHBvcnQgKiBmcm9tICcuL2xvYWRpbmcvbG9hZGluZy5jb21wb25lbnQnO1xuZXhwb3J0ICogZnJvbSAnLi9sb2FkaW5nLXNjcmVlbi9sb2FkaW5nLXNjcmVlbi5jb21wb25lbnQnO1xuIl19