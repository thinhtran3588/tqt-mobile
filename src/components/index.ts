export * from './text/text.component';
export * from './root-layout/root-layout.component';
export * from './loading/loading.component';
export * from './loading-screen/loading-screen.component';
