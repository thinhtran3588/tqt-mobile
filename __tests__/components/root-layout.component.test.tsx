import 'react-native';
import React from 'react';
import {render, toJSON} from '@test-utils';
import {RootLayout, Text} from '@lib/components';

it('renders correctly', async () => {
  const {baseElement} = render(
    <RootLayout>
      <Text>Text</Text>
    </RootLayout>,
  );
  expect(toJSON(baseElement)).toMatchSnapshot();
});
