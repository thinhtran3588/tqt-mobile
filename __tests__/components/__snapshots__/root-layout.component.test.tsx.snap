// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <SafeAreaView
        style={
          Array [
            Object {
              "flex": 0,
            },
          ]
        }
      />
      <SafeAreaView
        style={
          Array [
            Object {
              "flex": 1,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              undefined,
              Object {
                "fontSize": 12,
              },
            ]
          }
        >
          Text
        </Text>
      </SafeAreaView>
    </View>
  </View>
</View>
`;
